### DISCUSSION TITLE
Can't enable PHP session compression

### DISCUSSION THREAD

**Discussion by rrrfialho** [phpredis/phpredis#2517] [Q&A] [answered]
Hi, 

I'm using a linux with php 8.1 with apache, and installed php-redis extension through pecl install redis

I've configured session compression on my php.ini , but as you can see on the pictures it doesn't shows on php.ini as enabled (all other parameters shows normally and get updated if I change in php ini). 

redis ini I'm using 

redis.session.compression = zstd
redis.session.compression_level = 3

I've made some tests, querying redis directly and sessions doesn't seams to be compressed

Below is part of my phpinfo configuration (tryied to install on both php 8.1 and php 7.4)

![image](https://github.com/phpredis/phpredis/assets/5687861/35435cc4-902b-4fb3-af2d-081c3d0b12a2)

![image](https://github.com/phpredis/phpredis/assets/5687861/764ca476-0778-4878-9313-6c073eb37de0)

Any clues?
thank you!

**Comment by yatsukhnenko** [comment, marked as answer]
@rrrfialho this feature is not released yet
